### PR TITLE
Get real-time data from financial server

### PR DESCRIPTION
**Pull request by donnapep**
Adds a new `type` property to the component to specify whether real-time or historical data should be fetched.

Fires `rise-financial-response` using the following format:

rise-financial-response

Fires `rise-financial-error` if there's a problem retrieving data from the financial server.

### CODE REVIEW

**Comment by donnapep** [comment]
@Rise-Vision/delivery Please review. Thx.

File: .gitignore [hunk 1]
```diff
@@ -2,4 +2,6 @@ node_modules/
 bower_components/
 config/config.js
 rise-financial-version.js
+rise-financial.html
```
Comment (stulees) [inline .gitignore]: @donnapep I don't think we should ignore these files. When a designer uses Bower or npm to pull in this component, those files will be missing. They need to be able to use an ES5 version.

**Review by stulees — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (donnapep) [inline .gitignore]: @stulees This is akin to when we were committing the `dist` folder, which created problems for us when we weren't even sure it was necessary. I think we're in a similar situation here. If a designer is already pulling this component in via Bower or similar, then I tend to think that it's not a stretch for them to run `gulp build`.

Given this, what do you think about leaving it out for now and adding them back in if it becomes an issue? It creates extra work for us by committing them.

Comment (settinghead) [inline .gitignore]: You can also publish your bower or npm package so that it can contain these built files while leaving them out of the git history.

Comment (stulees) [inline .gitignore]: @donnapep Not sure I follow how a designer running `gulp build` will resolve it. Can you clarify? I see this as a different situation to the `dist` folder. When a designer has created a project that uses this component, if we .gitignore these files, then these files will not be in the bower (or similar) package for their project.

File: rise-financial-es6.js [hunk 2]
```diff
@@ -83,8 +108,63 @@
...
+
+      return `select ${ fields.join( "," ) }`;
+    }
 
-      console.log( this._instruments );  // eslint-disable-line no-console
+    _getRealTimeData( instruments, fields = [] ) {
+      const realTime = this.$.realTime;
+
+      realTime.url = this._getRealTimeUrl();
+      realTime.params = this._getRealTimeParams( instruments, fields );
```
Comment (settinghead) [inline rise-financial-es6.js]: @donnapep I suppose by setting `url` and `params`, byutv will automatically establish a connection? Is that how it works?

File: .gitignore (hunk 1 above)
Comment (donnapep) [inline .gitignore]: @settinghead That sounds like a great idea! @stulees I think we should do that instead of committing those files to Github.

Comment (stulees) [inline .gitignore]: @settinghead Nice, didn't know that was possible. @donnapep Agreed, so long as we can publish to contain those files I'm good with ignoring in git.

File: rise-financial-es6.js (hunk 2 above)
Comment (donnapep) [inline rise-financial-es6.js]: @settinghead That's correct. It's what the [`auto` attribute](https://github.com/Rise-Vision/rise-financial/pull/6/files#diff-1c4ad2054a29419e2f296d322fd3b64eR8) does.

**Review by stulees — APPROVED** [review]

**Comment by stulees** [comment]
@donnapep Looks good 👍 but will also defer to @settinghead

**Comment by donnapep** [comment]
For the sake of expediency, I'm merging this now. @settinghead If you have any changes, let me know and I'll put them in my next PR. Thanks @stulees and @settinghead.

**Comment by settinghead** [comment]
@donnapep no changes. Looks good. :+1:

**Pull request merged**